aspnetcore/fundamentals/apis.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "APIs overview"
+author: tdykstra
+description: Learn about differences between controller-based APIs and minimal APIs.
+ms.author: tdykstra
+ms.date: 01/10/2023
+uid: fundamentals/apis
+---
+
+# Choose between controller-based APIs and minimal APIs
+
+ASP.NET Core supports creating web APIs using controllers or using minimal APIs. *Controllers* in a web API project are classes that derive from <xref:Microsoft.AspNetCore.Mvc.ControllerBase>. *Minimal* refers to the minimal effort, code, and configuration that this approach requires for building fully functioning REST endpoints. This article points out differences between the two approaches.
```

Comment:
```suggestion
ASP.NET Core supports two approaches to creating APIs: a controller-based approach and minimal APIs. *Controllers* in a web API project are classes that derive from <xref:Microsoft.AspNetCore.Mvc.ControllerBase>. *Minimal APIs* are designed so that minimal code and configuration is required to create your REST endpoint. This article points out differences between the two approaches.
```

Reply:
Note: team would like to use "API" and "APIs" consistently rather than "web API" "REST endpoint" "HTTP endpoint" etc.

/cc @Rick-Anderson @tdykstra

Reply:
> is required to create **_your_** API

Better

is required to create **_an_** API